Keas.Mvc/ClientApp/src/components/Equipment/EquipmentEditValues.tsx

```diff
@@ -80,17 +80,17 @@ const EquipmentEditValues = (props: IProps) => {
 
   return (
     <div>
-      {props.disableEditing && props.openEditModal && (
-        <div className='row justify-content-between'>
-          <h3>Equipment Details</h3>
+      <div className='row justify-content-between'>
+        <h3>Equipment Details</h3>
```

Comment:
I don't know that this is needed if we are creating a new equipment.
It has the Create New Equipment title when we do that.
![image](https://github.com/ucdavis/Peaks/assets/469010/08700875-b80e-4617-9a77-b1fac1aec727)